session1/react16/src/js/components/pages/FragmentsExample.js

```diff
@@ -1,6 +1,6 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
```

Comment:
although, fragment here is not needed. Just in the following file.

Reply:
Totally, I was doing something there and forgot to remove it. Thanks